web/template/mlpa-documents.gohtml

```diff
@@ -18,7 +18,7 @@
                 </div>
 
                 <div class="moj-page-header-actions__actions">
-                    <div class="moj-button-menu">
+                    <div class="moj-button-menu" data-module="button-menu">
```

Comment:
This and payments don't need to be turned into a button menu, they just happen to have the `moj-button-menu` class, so you can remove the `data-module` attribute here and in `mlpa-payments.gohtml`